spec/api-native-image-spec.js

```diff
@@ -165,6 +165,10 @@ describe('nativeImage module', () => {
         { width: 538, height: 190, scaleFactor: 2.0 })
       expect(imageI.getSize()).to.deep.equal({ width: 269, height: 95 })
     })
+
+    it('throws on invalid arguments', () => {
+      expect(() => nativeImage.createFromBuffer(null)).to.throw('buffer must be a node Buffer')
```

Comment:
Can we add a test for something that conceivably be used to trick this API, like a normal array full of garbage data.  `[12, 14, 124, 12]`?.

Reply:
added